lib/ex_aws/dynamo_streams.ex

```diff
@@ -0,0 +1,117 @@
+defmodule ExAws.DynamoStreams do
+  @moduledoc """
+  Operations on DynamoDB Streams
+
+  http://docs.aws.amazon.com/dynamodbstreams/latest/APIReference/API_Operations.html
+
+  NOTE: When Mix.env in [:test, :dev] dynamo clients will run by default against
+  Dynamodb local.
+
+  Enabling/Disabling streams on a table is performed through the `Dynamo.update_table` operation.
+  The stream arn can then be retrieved with `Dynamo.describe_table` in the `LatestStreamArn` key.
+
+  """
+
+  import ExAws.Utils, only: [camelize_keys: 1, upcase: 1]
+  require Logger
+
+  @namespace "DynamoDBStreams_20120810"
+
+  ## Streams
+  ######################
+
+  @type stream_arn :: binary
+
+  @doc """
+  Lists all of the streams associated with an account.
+  If you have multiple streams on a table (created by disabling/enabling the stream)
+  you may have difficulty identifying the active stream from this operation.
+  """
+  @spec list_streams() :: ExAws.Operation.JSON.t
+  def list_streams() do
```

Comment:
The optional parameters should be supported from: http://docs.aws.amazon.com/dynamodbstreams/latest/APIReference/API_ListStreams.html